### ISSUE TITLE
How to split dataset into a training set, and a test set

### ISSUE THREAD

**Issue by kkang831** [genforce/idinvert#19]
Thank you for sharing the code.

To learn the encoder for ffhq dataset, I want to know how to split the dataset, and is there a code for split??

And, also, is there any pre processing for ffhq?

Thank you

**Comment by zhujiapeng** [comment]
We referred to the [dataset_tool.py](https://github.com/genforce/idinvert/blob/master/dataset_tool.py) offered by stylegan. We first get the `.png`  images from `.tfrecords` file and then save those `.png` images to the train `.tfrecords` and the test `.tfrecords`
